apps/nextra/generate-sitemap-index.cjs

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+const path = require('path');
+
+const generateSitemapIndex = () => {
+  const sitemapIndexContent = `<?xml version="1.0" encoding="UTF-8"?>
+  <sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <sitemap>
+      <loc>https://preview.aptos.dev/sitemap-en.xml</loc>
+    </sitemap>
+    <sitemap>
+      <loc>https://preview.aptos.dev/sitemap-zh.xml</loc>
+    </sitemap>
+    <sitemap>
+      <loc>https://preview.aptos.dev/sitemap-fr.xml</loc>
+    </sitemap>
+  </sitemapindex>`;
```

Comment:
Could we generate this by using fs to parse the pages directory and getting all subdirectories that are not `api`?